pyproject.toml

```
@@ -20,12 +20,8 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.7,<3.11"
-structlog = "^21.2.0"
-PyYAML = "^6.0.0"
-pydantic = "^1.9.0"
+python = ">=3.7,<3.12"
```

`airflow-ext` explicitly imports `structlog`, so I believe it should have a dependency on it, rather than relying on the EDK having a dependency on it. The dependency can be permissive to avoid conflicts:

```suggestion
python = ">=3.7,<3.12"
structlog = ">=20.1.0"
```